### ISSUE TITLE
Issue on page /12a_image_file_formats/opening_czi_files.html

### ISSUE THREAD

**Issue by tuhulab** [haesleinhuepf/BioImageAnalysisNotebooks#41]
The code shown in the notebook used `czifile`.
> we will open a .czi file using the [czifile library](https://pypi.org/project/czifile/).

But a later sentence contradicts with the previous statement.
> It is recommended to use [AICSImageIO](https://github.com/AllenCellModeling/aicsimageio) for reading CZI files **as shown in this notebook.**

Have I misunderstood anything?

**Comment by haesleinhuepf** [comment]
Good catch! The second sentence should contain a link to this page : https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/12a_image_file_formats/opening_images_using_aicsimageio.html

I'll fix that soon
